src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Index.cshtml

```diff
@@ -100,6 +100,8 @@
         </li>
         @if (Model.ShellSettingsEntries.Count > 0)
         {
+            var requestPath = FullRequestPath + (FullRequestPath.Contains('?') ? "&": "?");
```

Comment:
Remove

```suggestion
            
```

Reply:
Believe or not why I didn't think about this ;) may be someone used `FullRequestPath` for a reason